### ISSUE TITLE
Version 1.1.0 - Bug - Breakpoint not hit in debug mode

### ISSUE THREAD

**Issue by jams4code** [microsoft/vscode-azurefunctions#2609]
I had an issue while debugging my backend in local lately, so I'm posting the problem and the workaround in case you have the same issue:

**Issue: When running my backend in local, the application wasn't hitting my breakpoints anymore. => Impossible to debug**
 
Problem: Last release of the new version for azure-functions-core-tools@3, 5 days ago. The last version (1.1) has a bug I think.
 
_Workaround: You can go on your extensions manager: and select the precedent working version (1.0.1) for Azure Function._

**Comment by ejizba** [comment]
Please add more information specific to your situation that might help us reproduce. For example - your OS, the project's language, the exact version of the func cli, any errors you see, etc.

**Comment by jams4code** [comment]
Hi,
Thanks for your response.

I'm going to close the bug issue because now it seems to be working.
This issue was happening to different persons on my team (at least 4 of them confirmed that they had the same issue) when they had the update of their `ms-azuretools.vscode-azurefunctions` to V1.1.0.
So all the rest of the team did a rollback to the V1.0.1. And then it was working.

After reading your message this morning, I've tried, and everything is working smoothly. Maybe a dependency needed for the new version was not set correctly at the time or something else... (maybe related to the update of the net framework on my OS)

For information, we were all running on W10 Enterprise Build 17763; The project language is C#, and we did not have any errors. It was only impossible to stop on breakpoints. 

Kind regards,
Jamal
